l10n_be_intrastat_product/tests/test_intrastat_be.py

```diff
@@ -15,6 +15,7 @@ def setUpClass(cls):
 
         cls.company = cls.env.company
         cls.env.company.country_id = cls.env.ref("base.be")
+        cls.env.company.account_fiscal_country_id = cls.env.ref("base.be")
```

Comment:
What's do you declare this variable ? <br/>

Reply:
`account_fiscal_country_id` is a compute readonly=False that defaults to country_id.

TBH, I'm not entirely sure why it started to fail without this. It failed on a consistency error in `_validate_taxes_country` on the move reversal here:

https://github.com/OCA/l10n-belgium/blob/f6aa3907845b59a4b8dff6e3efd38d106f65c5ac/l10n_be_intrastat_product/tests/test_intrastat_be.py#L154-L166

I suspect it passed by accident before.

Reply:
Ideally these `l10n_be_intrastat_product` tests should use a new company with the BE CoA instead of hacking the country of the main company and changing the country on all the taxes as it does today.